configuration/stub-frontend-fed-config/special-cases/abc-variants.yml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
The special-cases folder will no longer be needed, I and John have removed this see https://github.com/alphagov/verify-frontend-federation-config/commit/8e1dfee16e077efe63936ec2d3bdd165a6a35db0

Reply:
Could we make a list of these changes and then can we raise a ticket?  I accept you're making breaking changes to the frontend and how it's configured but until those land I can't make the changes here without breaking verify-local-startup.  The other ideal is we land this and then you and John can make the adjustments and submit a PR ;-)